requirements/base.txt

```diff
@@ -149,7 +149,7 @@ django-capture-tag==1.0
     # via -r requirements/base.in
 django-colorfield==0.9.0
     # via -r requirements/base.in
-django-cookie-consent==0.4.0
+django-cookie-consent==0.5.0b0
```

Comment:
Is it a safe approach to use beta versions of a library? I saw that you know-have implemented this but just asking..